src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+const GET_ROCKET = 'spaceTravellerHub/rockets/GET_ROCKET';
+
+export const getRocket = (payload) => ({ type: GET_ROCKET, payload });
+
+const url = 'https://api.spacexdata.com/v3/rockets';
+
+export const fetchRocket = () => async (dispatch) => {
+  const response = await axios.get(url);
+  const data = await response.data;
+  const rockets = [];
+  data.forEach((rocket) => {
+    rockets.push({
+      id: rocket.id,
+      name: rocket.rocket_name,
+      description: rocket.description,
+      flickr_images: rocket.flickr_images[0],
+    });
```

Comment:
[OPTIONAL] Kindly use destructuring according to [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).